Use a seperate bucket to store AWS Config Configuration History, enable KMS on the delivery channel objects, and add the option to set a optional path for all supported IAM resources. (#164)
schubergphilis/terraform-aws-mcaf-landing-zone: UPGRADING.md

# Upgrading to 0.23.x

Version `0.23.x` introduces a change in behaviour of AWS Config:

> - By default the `aggregator_regions` were set to eu-west-1 and eu-central-1, this has been changed to only enable the current region. Please provide a list of regions to `var.aws_config.aggregator_regions` if you want to enable AWS Config in multiple regions.

Reviewer: ```suggestion
- By default the `aggregator_regions` were set to eu-west-1 and eu-central-1, this has been changed to only enable the current region. Provide a list of regions to `var.aws_config.aggregator_regions` if you want to enable AWS Config in multiple regions.
```

> - The `aws-controltower-logs` bucket was used to store the CloudTrail and AWS Config logs. This version introduces a seperate bucket for AWS Config. You are able to override the bucket name to comply with naming rules of your organisation.
> 

Reviewer: ```suggestion
- Previously the `aws-controltower-logs` bucket was used to store CloudTrail and AWS Config logs, this version introduces a separate bucket for AWS Config. You are able to override the bucket name by setting `var.XXX`.
```

Can you add how to override pls?
